src/action/wallet.js

```diff
@@ -181,18 +204,48 @@ class WalletAction {
    * screen.
    * @param  {string} options.walletPassword The user chosen password
    * @param  {Array}  options.seedMnemonic   The seed words to generate the wallet
+   * @param  {number} options.recoveryWindow The number of addresses to recover
    * @return {Promise<undefined>}
    */
-  async initWallet({ walletPassword, seedMnemonic }) {
+  async initWallet({ walletPassword, seedMnemonic, recoveryWindow = 0 }) {
     try {
       await this._grpc.sendUnlockerCommand('InitWallet', {
         wallet_password: toBuffer(walletPassword),
         cipher_seed_mnemonic: seedMnemonic,
+        recovery_window: recoveryWindow,
       });
       this._store.walletUnlocked = true;
       this._nav.goSeedSuccess();
     } catch (err) {
-      this._notification.display({ msg: 'Initializing wallet failed', err });
+      this._notification.display({
+        type: 'error',
+        msg: `Initializing wallet failed: ${err.details}`,
+      });
```

Comment:
The above does the same in one line. Why the change?

Reply:
Sorry, did you see this comment? https://github.com/lightninglabs/lightning-app/pull/612#discussion_r215411823 This is the reason I changed it, you may not be getting the same issue?